### DISCUSSION TITLE
Option to disable initial canvas focus

### DISCUSSION THREAD

**Discussion by jackharrhy** [replit/kaboom#753] [Ideas]
It would be nice if it was possible to disable the default functionality of focusing on the initial canvas

https://github.com/replit/kaboom/blob/4c0445e49302c8b893cafec13b54e619c8b8eb98/src/kaboom.ts#L6937

While this functionality makes sense for a full screen game, I'm planning on building a game that has non kaboom UI initially, but still has a kaboom canvas in the backdrop.

Currently I can get around this by creating a canvas manually, passing it in via the `canvas` property on `kaboom`, and after `kaboom`, calling `canvas.blur()` to remove the focused added by kaboom, but it would be nice for the canvas to not even be focused to begin with.

**Comment by slmjkdbtl** [comment]
Added `kaboom({ focus: false })` in latest versions
